src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -236,7 +234,7 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
       phoneNumber,
     } = address
     const usOrCanada = country === "US" || country === "CA"
-    return {
+    const errors = {
```

Comment:
This validator  is  slightly different  from the one shared by the payment forms, so decided to keep it in this file.